apps/explorer/src/components/object/FieldItem.tsx

```diff
@@ -58,7 +58,7 @@ export function FieldItem({
     }
 
     return (
-        <div className="truncate text-body-md text-neutral-40">
+        <div className="break-all text-body-md text-neutral-40">
```

Comment:
why did you use this instead of break-words?

Reply:
I used `break-all` because the strings displayed in this component are long hashes or identifiers, such as `655538b04f26288903f1d9fe95af50d82e5b55b22cd3a0128497c2a9603fc7de::stardust_upgrade_label::STARDUST_UPGRADE_LABEL` and these don’t have spaces or natural places to break, so `break-words` wouldn’t handle them well and could cause the text to overflow :pray: :see_no_evil: